src/init/mod.rs

```diff
@@ -121,68 +121,50 @@ pub fn init_stub(shell_name: &str) -> io::Result<()> {
+        ),
+        "zsh" => print!(
+            r#"source <("{}" init zsh --print-full-init)"#,
+            starship.sprint_posix()?
+        ),
+        "fish" => print!(
             // Fish does process substitution with pipes and psub instead of bash syntax
-            let script = format!(
-                "source (\"{}\" init fish --print-full-init | psub)",
-                starship.sprint_posix()?
-            );
-            Some(script)
-        }
-        "powershell" => {
-            // Explanation of syntax:
-            // &: Explicitly tells powershell to execute path with starship executable.
-            //
-            // @: multi-line stdout is returned as an array, but a single line or no lines
-            //    are returned as-is. @ ensures it's always an array.
-            //
-            // -join "`n": Joins the stdout array together as a string with newlines.
-            //             Powershell escapes with ` instead of \ thus `n translates to a newline.
-            let script = format!(
-                "Invoke-Expression (@(&\"{}\" init powershell --print-full-init) -join \"`n\")",
-                starship.sprint()?
-            );
-            Some(script)
-        }
-        "ion" => {
-            let script = format!("eval $({} init ion --print-full-init)", starship.sprint()?);
-            Some(script)
-        }
-        "elvish" => {
-            let script = format!(
-                "eval (\"{}\" init elvish --print-full-init | slurp)",
-                starship.sprint_posix()?
-            );
-            Some(script)
-        }
-        "tcsh" => {
-            let script = format!(
-                r#"eval "`("{}" init tcsh --print-full-init)`""#,
-                starship.sprint_posix()?
-            );
-            Some(script)
-        }
+            r#"source ("{}" init fish --print-full-init | psub)"#,
+            starship.sprint_posix()?
+        ),
+        "powershell" => print!(
+            /*
+             * Explanation of syntax:
+             * &: Explicitly tells powershell to execute path with starship executable.
+             *
+             * @: multi-line stdout is returned as an array, but a single line or no lines
+             *    are returned as-is. @ ensures it's always an array.
+             *
+             * -join "`n": Joins the stdout array together as a string with newlines.
+             *             Powershell escapes with ` instead of \ thus `n translates to a newline.
+             */
+            r#"Invoke-Expression (@(&"{}" init powershell --print-full-init) -join "`n")"#,
+            starship.sprint()?
+        ),
+        "ion" => print!("eval $({} init ion --print-full-init)", starship.sprint()?),
+        "elvish" => print!(
+            r#"eval ("{}" init elvish --print-full-init | slurp)"#,
+            starship.sprint_posix()?
+        ),
+        "tcsh" => print!(
+            r#"eval `("{}" init tcsh --print-full-init)`"#,
```

Comment:
Previously it looked like this:
```rust
r#"eval "`("{}" init tcsh --print-full-init)`""#,
```
I removed quotes around (back)ticks because they are redundant.